<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSetupForm</class>
 <widget class="QWidget" name="SystemSetupForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="systemCoordinatesGroup">
     <property name="title">
      <string>System Coordinates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="localFileLabel">
               <property name="text">
                <string>Local File</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="localFile"/>
               </item>
               <item>
                <widget class="QPushButton" name="localFileDialogButton">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>29</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>or</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="pdbEntryLabel">
            <property name="text">
             <string>PDB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pdbEntry">
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filterGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="chainsGroup">
        <property name="title">
         <string>Chains</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="chainsGroupLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="hetatmGroup">
        <property name="title">
         <string>Hetero Atoms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="systemSetupGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Solvation Box</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="forceFieldLabel">
            <property name="text">
             <string>Force Field</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="forceField"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="waterModelLabel">
            <property name="text">
             <string>Water Model</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="waterModel"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="boxTypeLabel">
            <property name="text">
             <string>Box Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxType">
            <property name="currentText">
             <string/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="distancetoEdgeLabel">
            <property name="text">
             <string>Distance to Edge (nm)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="distanceToEdge">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
